plugins/inputs/jolokia/jolokia.go

```diff
@@ -14,7 +14,16 @@ import (
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
-type Server struct {
+type JolokiaServer struct {
+	Name     string
+	Host     string
+	Username string
+	Password string
+	Port     string
+	Https    bool
+}
```

Comment:
I would agree with you, if there wouldn't be that proxy mode for this plugin.
But still, i will have a look at it.